lib/tests/streamlit/spinner_test.py

```diff
@@ -26,3 +26,7 @@ def test_spinner(self):
             time.sleep(0.2)
             el = self.get_delta_from_queue().new_element
             self.assertEqual(el.spinner.text, "some text")
+        # Check that the element gets reset to an empty container block:
+        last_delta = self.get_delta_from_queue()
+        self.assertTrue(last_delta.HasField("add_block"))
+        self.assertFalse(last_delta.add_block.allow_empty)
```

Comment:
This assertion is unintuitive ("allow_empty should be false for an empty block??") - do you mind adding a comment to the line saying why we expect this to be false?

Reply:
Addded a comment